HYCOM-interface/CMakeLists.txt

```diff
@@ -9,7 +9,11 @@ if(CMAKE_Fortran_COMPILER_ID MATCHES "GNU")
   set(CMAKE_Fortran_FLAGS_DEBUG "-O0 -fcheck=bounds -ffpe-trap=invalid,zero,overflow,underflow" )
   set(CMAKE_Fortran_LINK_FLAGS "")
 elseif(CMAKE_Fortran_COMPILER_ID MATCHES "Intel")
-  set(CMAKE_Fortran_FLAGS "-g -traceback -xSSE4.2 -mcmodel=small -r8")
+  if(SUPAFAST)
```

Comment:
@SamuelTrahanNOAA please make this change (removing -xSSE4.2) for all intel compiler build options, not only just under SUPAFAST option. In other words, I think the if (SUPAFAST) condition is not needed here. Thanks!

Reply:
Testing this now.